packages/lesswrong/server/callbacks/userCallbacks.ts

```diff
@@ -195,3 +198,65 @@ getCollectionHooks("Users").editSync.add(async function usersEditCheckEmail (mod
   }
   return modifier;
 });
+
+getCollectionHooks("Users").editAsync.add(async function subscribeToForumDigest (newUser: DbUser, oldUser: DbUser) {
+  if (
+    isAnyTest ||
+    forumTypeSetting.get() !== 'EAForum' ||
+    newUser.subscribedToDigest === oldUser.subscribedToDigest
+  ) {
+    return;
+  }
+
+  const mailchimpAPIKey = mailchimpAPIKeySetting.get();
+  const mailchimpForumDigestListId = mailchimpForumDigestListIdSetting.get();
+  const { lat: latitude, lng: longitude } = userGetLocation(newUser);
+  const status = newUser.subscribedToDigest ? 'subscribed' : 'unsubscribed';   
+  await fetch(`https://us8.api.mailchimp.com/3.0/lists/${mailchimpForumDigestListId}/members`, {
+    method: 'POST',
+    body: JSON.stringify({
+      email_address: newUser.email,
+      email_type: 'html', 
+      location: {
+        latitude,
+        longitude,
+      },
+      merge_fields: {
+        FNAME: newUser.fullName
+      },
+      status,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `API_KEY ${mailchimpAPIKey}`,
+    },
+  });
+});
+
+getCollectionHooks("Users").newAsync.add(async function subscribeToDripCampaign(user: DbUser) {
+  if (isAnyTest || forumTypeSetting.get() !== 'EAForum') {
+    return;
+  }
+  const mailchimpAPIKey = mailchimpAPIKeySetting.get();
+  const mailchimpEAForumListId = mailchimpEAForumListIdSetting.get();
```

Comment:
Exit early if these aren't set.

Reply:
Added in https://github.com/centre-for-effective-altruism/EAForum/pull/176/commits/72ec4223d4a9a31932b74460bb0a46f3d534e56e